### DISCUSSION TITLE
댓글 수정으로 비밀댓글 상태를 토글할까?

### DISCUSSION THREAD

**Discussion by zereight** [darass-team/2021-darass#729] [Ideas]
댓글 수정기능을 통해서는 현재 "내용"만 수정가능.

비밀댓글 -> 일반댓글
일반댓글 -> 비밀댓글

으로 수정할 수 있도록, 댓글 수정 시 따로 UI를 표시할지?

**Comment by pjy1368** [comment]
토글 찬성~

**Comment by jujubebat** [comment]
저도 토글 좋다고 생각합니다~~
